.github/workflows/publish-rock.yaml

```diff
@@ -0,0 +1,73 @@
+name: Pack Rock and Publish Image
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  pack-rock:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+
+      - name: Build assets
+        run: |
+          yarn install
+          yarn run build
+
+      - name: Setup LXD
+        uses: canonical/setup-lxd@main
+
+      - name: Setup rockcraft
+        run: sudo snap install rockcraft --classic --channel=latest/edge
```

Comment:
Forwarding [this](https://github.com/canonical/anbox-cloud.io/pull/352#discussion_r1732410572) question from @edlerd - what should we use? `stable` or `edge`? I see that kernel factory uses [`stable`](https://github.com/canonical/kernelfactory.canonical.com/blob/main/.github/workflows/deploy.yaml#L65) (CC @jw-can), but `dqlite.io` uses [`edge`](https://github.com/canonical/dqlite.io/blob/main/.github/workflows/deploy.yaml#L53). :confused: 

Same question applies to the `charmcraft` snap a few lines above.

Reply:
KF had been using `edge` until very recently, since that was the only place where the flask extensions were available. However, the extensions have been merged some while now, so you could also go ahead with using `stable`, which imho would make sense.

Small warning though: There's an issue in current rockcraft in stable, which makes [packing the rock fail](https://github.com/canonical/kernelfactory.canonical.com/actions/runs/10573677661/job/29293631716)). The suggestion is to use an older version of rockcraft.

Reply:
Thanks both for your feedback! For now, it seems safe to keep using the `edge` channel then. Marking as resolved.